tests/e2e/lib.sh

```diff
@@ -276,7 +276,20 @@ deploy_sensor_via_operator() {
         --output-secrets - \
     | kubectl -n stackrox apply -f -
 
-    kubectl apply -n stackrox -f tests/e2e/yaml/secured-cluster-cr.yaml
+    if [[ -n "${COLLECTION_METHOD:-}" ]]; then
+	echo "Overriding the product default collection method due to COLLECTION_METHOD variable: ${COLLECTION_METHOD}"
+    else
+       die "COLLECTION_METHOD not set"
+    fi
```

Comment:
Not sure if it's all necessary. In my opinion, the following would be sufficient:

```bash
if [[ -z "${COLLECTION_METHOD}" ]]; then
  die "COLLECTION_METHOD environment variable must be set"
end
```

Up to you.

Reply:
I will keep it as is for now.